thoth/common/workflows.py

```diff
@@ -0,0 +1,234 @@
+# thoth-common
+# Copyright(C) 2019 Marek Cermak
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Workflow management for Thoth."""
+
+import logging
+import json
+import random
+import re
```

Comment:
Haven't check all imports, some of them look unused - like this `re` import.

Reply:
Ah, correct, leftover from moving the helper function to helpers module :) Thanks!